api/buckets/pb/buckets.proto

```diff
@@ -19,6 +19,7 @@ message ListRequest {}
 
 message ListReply {
     repeated Root roots = 1;
+    int64 size = 2;
```

Comment:
the total size of all buckets was useful during tests but didn't really make sense on the rootreply object. moved it here to accompany a request for all buckets. let me know if that seems okay

Reply:
Nice, makes sense to me